src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1431,14 +1431,14 @@ public XMPPError onRemoveSource(JingleSession sourceJingleSession,
      *
      * @param sourceJingleSession source Jingle session from which SSRCs are
      *                            being removed.
-     * @param ssrcsToRemove the {@link MediaSSRCMap} of SSRCs to be removed from
+     * @param ssrcsToRemove the {@link MediaSourceMap} of SSRCs to be removed from
      *                      the conference.
      * @param updateChannels tells whether or not SSRC update request should be
      *                       sent to the bridge.
      */
     private void removeSSRCs(JingleSession        sourceJingleSession,
```

Comment:
please rename this method to removeSources and any mentions of SSRCs in JavaDoc

Please take a look on other modified methods as well